Sources/ForemWebView/ForemWebView.swift

```diff
@@ -4,7 +4,7 @@ import UIKit
 import WebKit
 import AVKit
 
-public protocol ForemWebViewDelegate: class {
+public protocol ForemWebViewDelegate: AnyObject {
```

Comment:
A recent Swift minor version suggested to change `class` with `AnyObject` here from a deprecation notice

```
Using 'class' keyword for protocol inheritance is deprecated; use 'AnyObject' instead
```